docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md

```diff
@@ -0,0 +1,210 @@
+
+You can create a universe using the provider as follows:
+
+- Use the Yugabyte Platform web UI to navigate to **Universes** and then click **Create Universe**. 
+
+- Complete the **Create Universe** page shown in the following illustration by entering the following information:
+
+  - In the **Name** field, enter universe-1.
+  - In the **Provider** field, enter ocp-test.
+  - In the **Regions** field, enter US East.
+  - In the **Instance Type** field, enter xsmall (2 cores, 4GB RAM).
+
+![img](https://lh3.googleusercontent.com/E8EwHd0olx_ZmmpRN-vpem8TAvqqU_gPQGyuMJH9yDxKS89cswTURFASkAE0fDKWMCfImUknxbyN7z87SzaOJJE_PO6Wo4aKw7UHIdXy7oCxPgTmv2XvRMvAIvbRQygS0gz3oXNz)
+
+- Click **Create**. 
+
+The following illustration shows the universe creation progress:
+
+![img](https://lh3.googleusercontent.com/wmVq3lUnqseriSgXcw-gwqNRct4CxUSZ0NnGwgVJ5UG-7GU8Ja4kKHGUZ428BmYOZvcemMLlod_9PBWQAFaj99bOybyW_XEc40X6mZL9fZKi6IQ9q7-Cr52XTfzjmw0ppGTN8b9Z)
+
+If you click universe-1 and then navigate to **Tasks**, you should see a page similar to the one shown in the following illustration:
+
+![img](https://lh3.googleusercontent.com/UFeWUHRm7Amxek1U8yxqeV3z4RiUll05Vo9VowiCImlELaCabHyaDFFT81XYbbBXUMoDAK8g13q9FP4ZbwOSAoCzsqGV83zqF9Jkcw_3pb4U4WxgC7MOiULbCpAArxIqSHMrDDIC)
+
+Upon successful creation of the universe, the **Overview** tab of universe-1 should look similar to the following:
+
+![img](https://lh5.googleusercontent.com/TpdExY6idrjH64UXfXKsY9RRwPaLU5OWfNHD_VZxrzRDWAXmG_dftROfNyxRX7TNWr7BgWxrsuZXOa4N_4KP9RHt4c9AgleCGSmziKudtIbqzrJAqOu7YL0oa9MpfdCmBvNE7rPL)
+
+## Troubleshooting the Universe Creation
+
+If the universe creation remains in Pending state for more than 2-3 minutes, open the OCP web console, navigate to **Workloads > Pods** and check if any of the pods are in pending state, as shown in the following illustration:
+
+![img](https://lh5.googleusercontent.com/jHJnEpQ6baGDDuppWND7O6z2SZM4dOPkIFekGa556xUffhOjEFdkM3EUvL2BEQXponEkEjYOwIFmPKu3z35-d903Vhz0D2i_6MyZEXstkrHN1jpgQQFgFy675cNwyzhonpGf65l9)
+
+Alternatively, you can execute the following command to check status of the pods:
+
+```shell
+oc get pods -n yb-platform -l chart=yugabyte
+# output
+NAME          READY STATUS  RESTARTS AGE
+yb-master-0   2/2   Running  0     5m58s
+yb-master-1   2/2   Running  0     5m58s
+yb-master-2   0/2   Pending  0     5m58s
+yb-tserver-0  2/2   Running  0     5m58s
+yb-tserver-1  2/2   Running  0     5m58s
+yb-tserver-2  2/2   Running  0     5m58s
+```
+
+If any of the pods are in pending state, perform the following: 
+
+- Login with an admin account and navigate to **Compute > Machine Sets**.
+- Open the Machine Set corresponding to your zone label (us-east4-a). 
+- Click **Desired Count** and increase the count by 1 or 2, as shown in the following illustration.
+- Click **Save**.
+
+![img](https://lh3.googleusercontent.com/N7ZHbdmANOWytA4Byyxd5P93Kq51t9QSI9Nr0xSK1YPlBTJOHpbKuz6NDwYlmV2v02ZE_k8F2Xx85KFWQx8mAldYb9TUT01M7Gf3jJMVmlbQdK6_9apgyCr7s8_XF3iWEvVFl0OO)
```

Comment:
ocp-yugabyte-platform-tshoot-2.png

Reply:
I did replace all the images (they no longer point to google). This change will be visible when I submit the PR.